docs/rules/arrow-body-style.md

```diff
@@ -8,12 +8,12 @@ This rule can enforce the use of braces around arrow function body.
 
 ## Options
 
-The rule takes one option, a string, which can be:
+The rule takes one or two options. The first is a string, which can be:
 
 * `"always"` enforces braces around the function body
 * `"as-needed"` enforces no braces where they can be omitted (default)
 
-### "always"
+The second one is an object for more fine-grained configuration when the first option is `"as-needed"`. Currently, the only available option is `requireReturnForObjectLiteral`, a boolean property. It's `false` by default. If set to `true`, it requires braces and an explicit return for object literals.
```

Comment:
>it requires braces and an explicit return for object literals.

_it requires braces for the function body and an explicit `return` when the return value is just an object literal_